Remove Trace.Fail in Swarm.ProcessMessageAsync (#641)
planetarium/libplanet: CHANGES.md

 -  Marked `Address` and `HashDigest` as readonly.  [[#610]]
 -  `IceServer.CreateTurnClient()` became to throw `ArgumentException` when
    received invalid url.  [[#622]]
 -  `Swarm<T>.ProcessMessageAsync()` became to throw `InvalidMessageException`

>     when received incomprehensible message.  [[#628], [#641]]

Reviewer: Since `Swarm<T>.ProcessMessageAsync()` is a private method, it doesn't need to be written to the changelog.

Author: Then, is the changelog needed? 🤔 
I'm not sure why(how) we should mention #628 on bug-fix of private method.

Reviewer: I think this doesn't need to be written to the changelog.

Reviewer: How about this?
```
Fixed bug that message processing is halted when received an invalid type of message.
```

Author: Awesome 😎 . But the issue looks like described it can make whole processes be halted.
I will write changelog again following your format, thanks.

    connection through a TURN relay server. [[#453], [#599]]
 -  Fixed a bug that `KeyNotFoundException` occurred when sending a message
    through the TurnClient.

>  -  Fixed bug that whole processes can be halted when received an invalid

Reviewer 2: ```suggestion
 -  Fixed bug that whole processes could halt when received an invalid
```
